README.md

```diff
@@ -52,6 +53,30 @@ Anyone who wants to discuss the direction of the project, design and implementat
 * [Current agenda and past meeting notes](https://docs.google.com/document/d/1exd8_IG6DkdvyA0eiTtL2z5K2Ra-y68VByUUgwP7I9A/edit?usp=sharing)
 * [Past meeting recordings](https://www.youtube.com/playlist?list=PLP0uDo-ZFnQP6NAgJWAtR9jaRcgqyQKVy)
 
+## Project Status
+
+The status of each storage provider supported by Rook can be found in the table below.
+Each API group is assigned its own individual status to reflect their varying maturity and stability.
+More details about API versioning and status in Kubernetes can be found on the Kubernetes [API versioning page](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning), but the key difference between the statuses are summarized below:
+
+* **Alpha:** The API may change in incompatible ways in a later software release without notice, recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of long-term support.
+* **Beta:** Support for the overall features will not be dropped, though details may change. Support for upgrading or migrating between versions will be provided, either through automation or manual steps.
+* **Stable:** Features will appear in released software for many subsequent versions and support for upgrading between versions will be provided with software automation in the vast majority of scenarios.
+
+
+Name | Details | API Group | Status
+-----|---------|-----------|-------
+Rook Framework|The framework for common storage specs and logic used to support other storage providers.|rook.io/v1alpha2|Alpha
+Ceph|[Ceph](https://ceph.com/) is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters.|ceph.rook.io/v1beta1|Beta
```

Comment:
I think I agree with that implication, but I do prefer the Rook framework API group to be alpha since it is so new still.

Reply:
I think that's fine. The rook framework API group is really an implementation detail of the other groups, so it should be fine that Ceph treats it as beta and under the covers it can change if needed.